src/sctools/groups.py

```diff
@@ -0,0 +1,160 @@
+"""
+Group QC outputs
+
+"""
+
+from crimson import picard
+import argparse
+import os
+import pandas as pd
+
+
+def aggregate_picard_metrics_by_row(file_names, output_name):
+    """
+    pipeline outputs picard QC metrics at single cell/sample level.
```

Comment:
The rest of SCTools uses numpy docblocks, can you convert these?

Reply:
changed